programs/token-metadata/program/src/instruction/mod.rs

```diff
@@ -585,7 +585,7 @@ pub enum MetadataInstruction {
     #[account(5, name="update_authority", desc="Update authority for the metadata account")]
     #[account(6, name="system_program", desc="System program")]
     #[account(7, name="sysvar_instructions", desc="Instructions sysvar account")]
-    #[account(8, name="spl_token_program", desc="SPL Token program")]
+    #[account(8, optional, name="spl_token_program", desc="SPL Token program")]
```

Comment:
We're just realizing this broke SemVer for the Rust client. Is there a reason this can't still be required?